src/mem/cache/prefetch/Prefetcher.py

```diff
@@ -1025,6 +1026,14 @@ class L2CompositeWithWorkerPrefetcher(CompositeWithWorkerPrefetcher):
     cxx_header = "mem/cache/prefetch/l2_composite_with_worker.hh"
 
     cdp = Param.CDP(CDP(is_sub_prefetcher=True), "")
+    cmc = Param.CMCPrefetcher(CMCPrefetcher(is_sub_prefetcher=True), "")
```

Comment:
Did you test the performance of putting cmc and bop in L2? Code without testing should not be merged

Reply:
Several tests I have done before all use this config
![Screenshot from 2024-10-10 16-41-40](https://github.com/user-attachments/assets/729c10fc-1f55-4ecf-8b57-9db827758ed0)